kinesumer.go

```diff
@@ -132,6 +132,30 @@ func (k *Kinesumer) GetShards() (shards []*kinesis.Shard, err error) {
 	return
 }
 
+func (k *Kinesumer) LaunchShardWorker(shards []*kinesis.Shard) (int, error) {
+	perm := rand.Perm(len(shards))
```

Comment:
Call `Seed` first. Or even better, create a `NewSource` and use that instead if the global one.